openeo/extra/job_management.py

```diff
@@ -76,7 +76,7 @@ def start_job(
     """
 
     def __init__(
-        self, poll_sleep: int = 60, root_dir: Optional[Union[str, Path]] = "."
+        self, poll_sleep: int = 60, root_dir: Optional[Union[str, Path]] = ".", max_running_duration: Optional[int] = 720
```

Comment:
I think in all other places we specify timeouts/delays in seconds, so it might be better for consistence to define `max_running_duration` in seconds too. Note that you can specify the default as `12 * 60 * 60` to make it more obvious that this is 12 hours

Reply:
done